pkg/cmd/run.go

```diff
@@ -127,6 +132,18 @@ doppler run --command "YOUR_COMMAND && YOUR_OTHER_COMMAND"`,
 				}
 			}
 
+			if addKey && preserveEnv {
+				// skip key if environment variables already contain this key
+				envKey := fmt.Sprintf("%s=", name)
```

Comment:
Could this introduce a bug where if you are trying to exclude `E` then `APPLE` will also be excluded?

Reply:
Can we instead get just the name of the env var and compare it to the secret name?

Reply:
I don't see how it could

Reply:
Ohh because you are doing a hasPrefix, it safeguards against that case.